bin/improver-threshold

```diff
@@ -128,9 +129,15 @@ def main():
                 thresholds_from_file = json.load(input_file)
             thresholds = []
             fuzzy_bounds = []
+            is_fuzzy = True
```

Comment:
As far as I can tell, the `is_fuzzy` variable can be removed, so the code change below would just be:
```python
                if thresholds_from_file[key] == "None":
                    fuzzy_bounds = None
                else:
                    fuzzy_bounds.append(tuple(thresholds_from_file[key]))
```

Reply:
I used a boolean `is_fuzzy` so that this `if` statement isn't triggered for every item in the dictionary.  If there are fuzzy bounds in the dictionary `fuzzy_bounds` is a list to append to, but if not it is `None` (rather than a list of `None`s) and just needs to be set once.

Reply:
OK, I see the purpose of the `is_fuzzy` check now. It might be worth adding a comment to say something like "If is_fuzzy is False, then subsequent checks for fuzzy bounds are skipped."